docs/developer/tutorials/snmp/profile-format.md

```diff
@@ -232,7 +232,50 @@ metrics:
         tag: interface
 ```
 
-- Using an "index", i.e. one of the values in the `INDEX` field of the table MIB definition:
+##### Using a column from a different table with different indexes
```

Comment:
This is a super nit, but would it make sense to move this section after `Using an "index", i.e. one of the values in the `INDEX` field of the table MIB definition` because the next section kind of explains indexes and how to identify and use them?

Reply:
```
Using an "index", i.e. one of the values in the `INDEX` field of the table MIB definition
```

This section is about another way of using the indexes, not necessarily related to the other section ^. At the moment, this section doesn't explain either what indexes are. In the doc, for now, we assume people are a bit familiar with SNMP table indexes. We can possibly add an explanation about what SNMP indexes are, but as another PR.